.travis.yml

```diff
@@ -24,7 +24,7 @@ env:
     - MOUNT_LOCAL_SOURCES="false"
     - AIRFLOW_MOUNT_HOST_VOLUMES_FOR_STATIC_CHECKS="true"
     - FORCE_ANSWER_TO_QUESTIONS="yes"
-    - AIRFLOW_CI_SILENT="true"
+    - AIRFLOW_CI_SILENT="false"
```

Comment:
It looks like unrelated changes.

Reply:
True. It was for debugging of a problem. Will revert.